### PR TITLE
Adds READMEs to the packages.

### PR DESCRIPTION
**Pull request by agalbachicar**
Signed-off-by: Agustin Alba Chicar <ag.albachicar@gmail.com>

# 🎉 New feature

Adds README files to the packages.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: maliput_ros/README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+It is recommended to follow the guidelines for setting up a development workspace as described [here](https://maliput.readthedocs.io/en/latest/developer_setup.html).
+
+## Contributing
+
+Please see [CONTRIBUTING](https://maliput.readthedocs.io/en/latest/contributing.html) page.
+
+## License
+
+[![License](https://img.shields.io/badge/License-BSD_3--Clause-blue.svg)](https://github.com/maliput/ros2_maliput/blob/main/LICENSE)
```
Comment (francocipollone) [inline maliput_ros/README.md:121]: It is pointint to repo's license, we should actually have a license per package(thinking on future releasing via ros build farm)

**Review by francocipollone — APPROVED** [review]
LGTM!

File: maliput_ros/README.md (hunk 1 above)
Comment (agalbachicar) [reply maliput_ros/README.md]: Good point.

**Review by francocipollone — APPROVED** [review]
LGTM!

**Pull request merged**